common/app/Panes/Panes.jsx

```diff
@@ -58,15 +58,15 @@ export class Panes extends PureComponent {
       panes
     } = this.props;
     return panes.map(({ name, left, right, dividerLeft }) => {
-      const divider = dividerLeft ?
-        (
+      const divider = dividerLeft
+        ? (
           <Divider
             key={ name + 'divider' }
             left={ dividerLeft }
             name={ name }
           />
-        ) :
-        null;
+        )
+        : null;
```

Comment:
The changes to this file are unnecessary and out of scope of the PR.

Reply:
I'm not sure what happened here. I must have saved the file and the linter auto-adjusted the syntax? How do I remove this file from the PR?

Reply:
You can use git checkout ./file.js

You'll need to revert that specific file. see: https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git